### ISSUE TITLE
Feature request: 'copy' module for target-to-target

### ISSUE THREAD

**Issue by jnojr** [ansible/ansible-modules-core#365]
##### ISSUE TYPE

Feature Idea
##### COMPONENT NAME

copy module
##### ANSIBLE VERSION

N/A
##### SUMMARY

It would be nice to have a 'copy' module where the source and destination can both be on the target host.  That way, one unusable command can copy/move, set ownership, permissions, SELinux context, etc. without having to use multiple command or shell directives.

**Comment by alex-deref** [comment]
I've written a module which does what you asked, just trying to determine if it belongs in Ansible:

https://gist.github.com/alex-deref/dc66a0ca7e1904c30508

**Comment by bcoca** [comment]
In current devel try with  remote_src=true

**Comment by ansibot** [comment]
@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
This repository has been locked. All new issues and pull requests should be filed in https://github.com/ansible/ansible

Please read through the <a href="http://docs.ansible.com/ansible/dev_guide/repomerge.html#move-issues-and-prs-to-new-repo">repomerge</a> page in the dev guide. The guide contains links to tools which automatically move your issue or pull request to the ansible/ansible repo.

**Comment by ansibot** [comment]
This issue was migrated to https://github.com/ansible/ansible/issues/30214
